Documentation/NuttX.html

```diff
@@ -1450,11 +1450,13 @@ <h2>Git Repository</h2>
 <h2>Released Versions</h2>
 <p>
   In addition to the ever-changing GIT repository, there are frozen released versions of NuttX available.
-  The current release is NuttX 8.2.
-  NuttX 8.2 is the 133<sup>rd</sup> release of NuttX.
-  It was released on November 16, 2019, and is available for download from the
-  <a href="https://bitbucket.org/nuttx/nuttx/downloads/">Bitbucket.org</a> website.
-  Note that the release consists of two tarballs:  <code>nuttx-8.2.tar.gz</code> and <code>apps-8.2.tar.gz</code>.
+  The current release is NuttX 9.0.
+  NuttX 9.0 is the 134<sup>rd</sup> release of NuttX.
+  It was released on MMMM DD, YYYY, and is available for download from the
```

Comment:
When do you think it's a good time to change this?

Reply:
@Ouss4 How far are we away from release? Do we have a target release date?

Reply:
We are aiming for the 30th of April.

Reply:
@Ouss4 Can we merge now, then update the website with the HTML once we release? Or does the HTML get uploaded to the website automatically?

Reply:
The html gets inlined as part of a build step and then these files are uploaded to the wiki after the release.

The website is a sperate task where a page is generated from the release notes an a little metadata and then built by the CI and deployed

Reply:
Thanks for the info @btashton. @Ouss4 Will you put in the 30 April 2020 date? Then we should be good to merge this PR, and can start working on the other things that are needed. If we need to adjust the date up or down later we can always create a one-line PR to fix the date.

Reply:
Done.

Reply:
@Ouss4 Thanks!

@btashton @hartmannathan @patacongo Will one of you merge this when the checks finish ok?